src/renderers/noop/ReactNoop.js

```diff
@@ -160,6 +172,14 @@ var ReactNoop = {
     }
   },
 
+  hasScheduledDeferredCallback() {
+    return Boolean(scheduledDeferredCallback);
```

Comment:
Just seems better to use the "public" APIs for testing when possible. Don't have a strong opinion on this though.